src/DynamoCore/Nodes/NodeCategories.cs

```diff
@@ -454,6 +454,147 @@ internal static string InsertSpacesToString(string original)
             return newText.ToString();
         }
 
+        /// <summary>
+        /// Gets words from text, e.g. ImportFromCSV to ("Import","From","CSV")
+        /// </summary>
+        /// <param name="text">incoming string</param>
+        /// <param name="maxCharacters">Max number of characters per row</param>
+        internal static IEnumerable<string> WrapText(string text, int maxCharacters)
+        {
+            List<string> words = new List<string>();
+            if (string.IsNullOrWhiteSpace(text))
+                return words;
```

Comment:
Does `string.IsNullOrWhiteSpace` catch the case where `text` is an empty string? If it doesn't, then the following `text[0]` will crash.